rules/rules.libsonnet

```diff
@@ -56,6 +56,51 @@
           },
         ],
       },
+      {
+        name: 'noobaa-odf.rules',
+        rules: [
+          {
+            record: 'odf_system_health_status',
+            expr: |||
+              NooBaa_health_status
+            ||| % $._config,
+            labels: {           
+                system_vendor: 'Red Hat',
+                system_type: 'OCS',
+            },
+          },
+          {
+            record: 'job:odf_system_raw_capacity_used_bytes',
+            expr: |||
+              NooBaa_system_capacity
```

Comment:
Do we want this or the sum of the providers logical?

Reply:
I am not sure, I think that this is the most fitting one.